src/components/RightSidebar.tsx

```diff
@@ -0,0 +1,70 @@
+import { AnimatePresence, motion } from "framer-motion";
+import { ReactNode, useState } from "react";
+import { Css } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconKey } from "./Icon";
+import { IconButton } from "./IconButton";
+
+export type SidebarContentProps = {
+  icon: IconKey;
+  render: () => ReactNode;
+};
+
+export type RightSidebarProps = {
+  content: SidebarContentProps[];
+};
+
+export function RightSidebar({ content }: RightSidebarProps) {
+  const [selectedIcon, setSelectedIcon] = useState<IconKey | undefined>(undefined);
+  const tid = useTestIds({}, "rightSidebar");
+  const width = 380;
+
+  return (
+    <>
+      <div css={Css.df.jcfe.relative.pr3.$}>
+        <div css={Css.df.gap2.z1.$}>
+          {content.map(({ icon }) => (
+            <IconButton
+              key={`${icon}-${selectedIcon}`}
```

Comment:
thought: since `selectedIcon` isn't a stable value (expected to change on selection), I would think you wouldn't want to include that as the `key` here - was this an intentional addition vs just `icon`?

Reply:
It unsets the `active` state of the button. Without it, the button stays gray after you select a different icon.

Reply:
Ah okay makes sense, can you add an in-source comment for future us about that?